src/Core/tests/DeviceTests/Handlers/SearchBar/SearchBarHandlerTests.Windows.cs

```diff
@@ -8,6 +8,34 @@ namespace Microsoft.Maui.DeviceTests
 {
 	public partial class SearchBarHandlerTests
 	{
+		[Theory(DisplayName = "CancelButtonColor Initializes Correctly")]
+		[InlineData(0xFFFF0000)]
+		[InlineData(0xFF00FF00)]
+		[InlineData(0xFF0000FF)]
+		public async Task CancelButtonColorInitializesCorrectly(uint color)
+		{
+			var expected = Color.FromUint(color);
+
+			var searchBar = new SearchBarStub()
+			{
+				Text = "Test",
+				CancelButtonColor = expected
+			};
+
+			searchBar.Focus();
+
+			await InvokeOnMainThreadAsync(async () =>
+			{
+				var searchBarHandler = CreateHandler(searchBar);
+				await searchBarHandler.PlatformView.AttachAndRun(async () =>
+				{
+					await AssertionExtensions.Wait(() => searchBarHandler.PlatformView.FocusState != UI.Xaml.FocusState.Unfocused);
```

Comment:
hmm... what is the reason for checking this?

Reply:
The `CancelButton` only appears when the SearchBar has text and the focus. For that reason, here, we wait until be sure that the native control is focused.